flow/cmd/mirror_status.go

```diff
@@ -185,7 +196,9 @@ func (h *FlowRequestHandler) cloneTableSummary(
 	var numRowsSynced pgtype.Int8
 	var avgTimePerPartitionMs pgtype.Float8
 
-	rows, err := h.pool.Query(ctx, q, "clone_"+flowJobName+"_%")
+	flowNamePart := strings.Replace("clone_"+
+		strings.Replace(flowJobName, "%", "\\%", -1)+"_\\%", "_", "\\_", -1)
```

Comment:
```suggestion
		strings.Replace(flowJobName, "%", "\\%", -1)+"\\_\\%", "_", "\\_", -1)
```
then move replace more nestedly